providers/sendgrid/src/lib/sendgrid.provider.spec.ts

```diff
@@ -1,31 +1,33 @@
 import sendgridMail from '@sendgrid/mail';
-
 import { SendgridEmailProvider } from './sendgrid.provider';
 
+const mocKconfig = {
+  apiKey: 'SG.1234',
+  from: 'test@tet.com',
+};
```

Comment:
Nice! Liked the reusable mock config

Reply:
I'll fix the typo

Reply:
@galezra Haven't even noticed it before you wrote XD

Reply:
Done :)